e2e/tests/nightly/admin/user-profile-admin.spec.ts

```diff
@@ -0,0 +1,469 @@
+    await new UserAuditPage(newPage2).waitForLoad();
+
+    await adminTab.bringToFront();
+    await userProfileAdminPage.waitForLoad();
+
+    // Check BACK (admin/user) link is working
+    await userProfileAdminPage.getBackLink().click();
+    await new UserAdminPage(adminTab).waitForLoad();
+  });
+
+  test('Can bypass Google 2-Step Verification module', async () => {
+    const userProfileAdminPage = new UserProfileAdminPage(adminTab);
+    await userProfileAdminPage.waitForLoad();
+
+    const accessStatusTable = await userProfileAdminPage.getAccessStatusTable();
+
+    // Last completed on is a dash
+    const lastCompletedOnCell = await accessStatusTable.getCellByValue(
+      AccessModules.GOOGLE_VERIFICATION,
+      TableColumns.LAST_COMPLETED_ON
+    );
+    const completedOn = await lastCompletedOnCell.getCellValue();
+    expect(completedOn).toEqual('-');
+
+    // Expires on is NEVER for Google 2-step verification module
+    const expiresOnCell = await accessStatusTable.getCellByValue(
+      AccessModules.GOOGLE_VERIFICATION,
+      TableColumns.EXPIRES_ON
+    );
+    const expiresOn = await expiresOnCell.getCellValue();
+    expect(expiresOn).toEqual('Never');
+
+    // Change Bypass for Google 2-step verification module
+    const statusCell = await accessStatusTable.getCellByValue(AccessModules.GOOGLE_VERIFICATION, TableColumns.STATUS);
+    const oldStatus = await statusCell.getCellValue();
+
+    const bypassSwitch = await userProfileAdminPage.getBypassSwitchForRow(AccessModules.GOOGLE_VERIFICATION);
+
+    if (oldStatus === 'Bypassed') {
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+      expect(await bypassSwitch.isOn()).toBe(true);
+      await bypassSwitch.turnOff();
+    } else {
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('No data access');
+      expect(await bypassSwitch.isOn()).toBe(false);
+      await bypassSwitch.turnOn();
+    }
+
+    const bypassSwitchCell = await accessStatusTable.getCellByValue(AccessModules.GOOGLE_VERIFICATION, 'Bypass');
+    await verifyToggleSwitchNewBackgroundColor(adminTab, bypassSwitchCell);
+
+    const saveButton = userProfileAdminPage.getSaveButton();
+    await verifyButtonEnabled(saveButton, true);
+    await saveButton.click();
+    await userProfileAdminPage.waitForLoad();
+
+    const newStatus = await statusCell.getCellValue();
+    if (oldStatus === 'Bypassed') {
+      expect(newStatus).toBe('Incomplete');
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('No data access');
+    } else {
+      expect(newStatus).toBe('Bypassed');
+      expect(await userProfileAdminPage.getDataAccessTiers()).toEqual('Registered Tier');
+    }
+
+    if (newStatus === 'Incomplete') {
+      await bypassSwitch.turnOn();
+      await saveButton.click();
+      await userProfileAdminPage.waitForLoad();
+    }
+  });
+
+  test('Can bypass Controlled Tier training module', async () => {
```

Comment:
curious - why these 2 modules?

Reply:
Hm - not sure I agree.  But we can have a conversation about that for a different PR.  For this one, could you add something like "chose these 2 arbitrarily"